pylogix/lgx_device.py

```diff
@@ -17,6 +17,12 @@
 import socket
 from struct import pack, unpack_from
 
+from pylogix.utils import is_micropython
+
+if not is_micropython():
```

Comment:
this should be with the **vendors = lgx_vendors.vendors** statement below.

Also, would the following be more simple and Pythonic?
```
try:
    assert is_micropython()
    vendors = dict()
    ### => No change needed to static method Device.get_vendor,
    ###    because it will return 'Unknown' by default
except:
    import lgx_vendors as vendors

Reply:
yes I like this try catch, however for micropython vendors isn't a dictionary, but yes it should be changed to the try catch for sure.